### DISCUSSION TITLE
xrdp doesn't perform well on Linux systems

### DISCUSSION THREAD

**Discussion by ChenpiDog** [neutrinolabs/xrdp#2791] [Q&A]
I installed xrdp on the Linux system (deepin 20.9), and then remotely connected to the Linux desktop through a win11 computer, but the operation was not smooth.

**Comment by ChenpiDog** [comment]
The interface response is obviously stuck.

**Comment by ChenpiDog** [comment]
There is another problem, if I have logged in a certain user on the Linux server, if my personal computer uses this user for remote connection, the screen will be black. Only when the user on the server remains at the stage of waiting for the password can I connect remotely normally.

**Reply by matt335672** [reply]
This is usual I'm afraid - see the [FAQ](/neutrinolabs/xrdp/wiki/Tips-and-FAQ#why-cant-i-log-the-same-user-on-on-the-graphical-console-and-over-xrdp-at-the-same-time).
